easybuild/easyblocks/t/tbb.py

```diff
@@ -171,9 +170,13 @@ def install_step(self):
             cand_lib_paths = glob.glob(os.path.join(self.installdir, 'build', '*_release'))
             if len(cand_lib_paths) == 1:
                 self.libpath = os.path.join('build', os.path.basename(cand_lib_paths[0]))
+                # and symlink the right lib from /tbb/libs/intel64/... to lib
```

Comment:
This comment is incorrect: the `and` start doesn't make sense, and it symlinks to a subdirectory of `build`?